src/auth.js

```diff
@@ -163,12 +163,12 @@ async function fetchUrl(url, opts = {}, accept, parse) {
  * @param {string} url
  * @param {object} opts
  */
-const getFileURL = (url, opts) =>
+const getFileURL = (url, opts, timestamp) =>
   new Promise((resolve, reject) => {
     const auth = sessionStorage.getItem("auth");
     if (auth == "true") {
       opts.headers = { ...opts.headers, ...getHeaders() };
-      fetch(url, opts).then((response) => {
+      fetch(timestamp ? `${url}?d=${timestamp}` : url, opts).then((response) => {
```

Comment:
please use `ct` (creation_timestamp) which will be new file attribute.

Reply:
Done